fastchat/serve/model_worker.py

```diff
@@ -164,17 +168,18 @@ def get_status(self):
     def generate_stream_gate(self, params):
         try:
             for output in self.generate_stream_func(
-                self.model,
-                self.tokenizer,
-                params,
-                self.device,
-                self.context_len,
-                args.stream_interval,
+                    self.model,
+                    self.tokenizer,
+                    params,
+                    self.device,
+                    self.context_len,
+                    args.stream_interval,
             ):
                 ret = {
                     "text": output,
                     "error_code": 0,
                 }
+                self.clear_torch_cache()
```

Comment:
what does this clear cache actually do?

Reply:
The clear_torch_cache function is designed to clear GPU memory cache used by the PyTorch library. This can be helpful in reducing memory overhead and avoiding out-of-memory errors when working with large models or datasets.